enterprise-beans/4.0/_index.md

```diff
@@ -6,19 +6,28 @@ summary: "Release for Jakarta EE 9"
 
 Jakarta Enterprise Beans defines an architecture for the development and deployment of component-based business applications.
 
-* [Jakarta Enterprise Beans 4.0 Release Record]()
-  * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
+* [Jakarta Enterprise Beans 4.0 Release Record](https://projects.eclipse.org/projects/ee4j.ejb/releases/4.0/plan)
+  * [Jakarta Enterprise Beans 4.0 Release Plan](https://github.com/eclipse-ee4j/ejb-api/blob/master/4.0-PLAN.adoc)
 * [Jakarta Enterprise Beans 4.0 Specification Document]() (PDF)
 * [Jakarta Enterprise Beans 4.0 Specification Document]() (HTML)
 * [Jakarta Enterprise Beans 4.0 Javadoc](./apidocs)
 * [Jakarta Enterprise Beans 4.0 TCK]()([sig](),[sha](),[pub]())
 * Maven coordinates
   * [jakarta.ejb:jakarta.ejb-api:jar:4.0.0]()
 
+The Jakarta Enterprise Beans 4.0 intended scope includes all plans detailed in the [Jakarta EE 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan), which in essence involves a backwards incompatible namespace change from `javax.ejb` to `jakarta.ejb`.  In addition the Jakarta Enterprise Beans 4.0 Release Plan will involve:
+
+ - Removal of methods relying on `java.security.Identity`
+ - Removal of methods relying on JAX-RPC
+ - Removal of deprecated `EJBContext.getEnvironment()` method
+ - Removal of Support for Distributed Interoperability
+ - Mark optional EJB 2.x API Group
+
+See the [Jakarta Enterprise Beans 4.0 Release Plan](https://github.com/eclipse-ee4j/ejb-api/blob/master/4.0-PLAN.adoc) for full details.
 
 # Compatible Implementations
 
-* [Jakarta Enterprise Beans]()
+* Pending
```

Comment:
If this isn't planned to be GlassFish, that would be good to know now,
since it would effect our platform release plans.